packages/react-storage/src/components/StorageBrowser/views/LocationActionView/useHandleUpload.ts

```diff
@@ -135,10 +138,20 @@ export function useHandleUpload({
         };
       });
 
+      tracker.recordTasksToProcess(newTasks);
       return mergeSelectedTasks(prevTasks, newTasks);
     });
   };
 
+  const handleCancelAll = React.useCallback(() => {
+    tasks.forEach(({ cancel }) => (isFunction(cancel) ? cancel() : undefined));
```

Comment:
This syntax feels odd it me. Can cancel be something other than a function?
```
cancel?.();
```

Reply:
This is the current implementation of `handleCancelAll` function on line 236 and the only change was conversion to useCallback. I believe cancel should always be a function.